index.html

```diff
@@ -0,0 +1,331 @@
+> - MINOR version when you add functionality in a backwards compatible manner
+
+Patch and pre-release qualifiers are judged to be not particularly meaningful in the context of specifications, which are (by definition) interfaces rather than implementations. The `‹patch›` component of version denotes a bug fix which is backwards compatible—that is, a change to the implementation which does not affect the interface. `‹patch›`-level changes in the version of a spec denote wording clarifications which do not require implementation changes. As such, it is not important to track them for the purposes of version resolution.
+
+As with [semver](https://semver.org/spec/v2.0.0.html), the `0.x` version series is special: there is no expectation of compatibility between versions `0.x` and `0.y`. For example, a processor must not activate implementation `0.4` to satisfy a requested version of `0.2`.
+
+### Ordering
+Versions can be ordered with the following pseudocode:
+
+```typescript
+compareVersions(a: Version, b: Version) -> LESS_THAN | EQUAL | GREATER_THAN {
+  if a.major < b.major {
+    return LESS_THAN
+  }
+  if a.major == b.major {
+    if a.minor == b.minor { return EQUAL }
+    if a.minor > b.minor { return GREATER_THAN }
+    return LESS_THAN
+  }
+  if a.major > b.major {
+    return GREATER_THAN
+  }
+}
+```
+
+Note that this ordering does *not* guarantee that if A > B then A can satisfy a version requirement of B. For example, `2.0 > 1.9`, but `2.0` cannot [satisfy](#satisfying-version-requirements) the requirements of a document which references version `1.9`, since the major version update introduced possibly-breaking changes.
+
+### Satisfaction
+
+Given a version `requested` by a document and an `available` version of an implementation, the following pseudocode will determine if the `available` version can satisfy the `requested` version:
+
+```typescript
+canSatisfy(requested: Version, available: Version) =
+  requested.major == available.major &&
+  (requested.major > 0 && (requested.minor <= available.minor)
+    || (requested.minor == available.minor))
+```
+
+### Referencing versions and activating implementations
+
+Schema documents **must** reference a spec version which supports all the schema elements and behaviors required by the document. As a practical matter, authors should prefer to reference the [lowest](#ordering) versions which can [satisfy](#satisfaction) their requirements, as this imposes the fewest constraints on processors and is thus more likely to be supported.
+
+For every spec they support, processors **must** activate an implementation which can [satisfy](#satisfaction) the version required by the document.
+
+### Backus-Naur Form for versions and version tags
+
+```plaintext
+#::[BNF grammar for SemVer specifiers]
+‹version tag› ::= "v" ‹version›
+
+‹version› ::= ‹major›
+            | ‹major› "." ‹minor›
+
+‹major› ::= ‹numeric identifier›
+‹minor› ::= ‹numeric identifier›
+‹numeric identifier› ::= "0"
```

Comment:
Is a `<numeric identifier>` == `<digits>`? It feels like an additional and unnecessary step since the `"0"` case is already handled within `<digit>` (and the repeating nature handled by `<digits>`).

Reply:
`<numeric identifier>` is there to exclude leading zeroes (e.g. `"0" "1"` is a valid `<digits>` but not a valid `<numeric identifier>`).

Reply:
ya. also, we just copied this from semver, so i trust it's been ironed out